package.yaml

```diff
@@ -19,6 +19,12 @@ extra-doc-files:
 
 extra-source-files:
   - embed/preflight.css
+  - test/resources/basic.txt
```

Comment:
Doesn't this add these text files to hackage? Why do we need them in extra-source-files when they are only used in tests?

Reply:
If you pull down the source from Hackage, then the tests fail without those files. Since I pull the web-view source from github with the flake, I don't need those files specifed in the cabal file.

Reply:
let me know if you still want to remove them from extra-source-files.

Reply:
I'd rather they be removed from the extra-source-files if it won't mess up your approach here.